content/server/configuring.md

```diff
@@ -82,6 +82,16 @@ app:
     API_KEY: thisissupersecret
 ```
 
+<a name="administrator"></a>
+## Administrator
+
+After launching Skygear Server, an administrative role (`Admin`) and an
+adminitrator is created. The default username is `admin` and the default
+password is `secret`.
+
+Please remember to change the password of the administrator before your
```

Comment:
@ben181231 I am thinking about including sample code in each SDK's getting started to change admin password.. if there are no better ways to do it